firmware/application/apps/ui_recon.hpp

```diff
@@ -145,7 +145,7 @@ class ReconView : public View {
         {{0 * 8, 0 * 16}, "LNA:   VGA:   AMP:  VOL:     ", Color::light_grey()},
```

Comment:
nit: not necessary for this PR but `bool fwd = {true};` should be either assignment `fwd = true;` or initialization `fwd{true};`. The rest of the codebase prefers the second. The rest of this file prefers the first.

Reply:
I didn't knew what was the difference, so I went and had a look.
= -> copy assignment
{} -> direct assignment
I changed all the assignments.